### PR TITLE
Day 4 add remove

### PR DESCRIPTION
**Pull request by erezfree29**
To do list a complete project 
-User able to add tasks
-User able to sort tasks
-A task name can be edited
-A task can be deleted 
-Multiple task can be marked as complete and then removed.

### CODE REVIEW

File: src/scripts/sorting.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    if (e.stopPropagation) {
+      e.stopPropagation(); // Stops some browsers from redirecting.
+    }
+
+    // Don't do anything if dropping the same column we're dragging.
+    if (dragSrcEl !== this) {
+      // Set the source column's HTML to the HTML of the column we dropped on.
+      // alert(this.outerHTML);
+      // dragSrcEl.innerHTML = this.innerHTML;
+      // this.innerHTML = e.dataTransfer.getData('text/html');
```
Comment (mimipeshy) [inline src/scripts/sorting.js]: - [ ] It’s better to delete commented code. If you need to review a previous version of your code, use git history.

File: src/scripts/sorting.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+          tasks[i].index = i;
+        }
+      }
+    }
+
+    localStorage.setItem('tasksListArray', JSON.stringify(tasks));
+
+    /* [].forEach.call(cols, function (col) {
+      col.classList.remove('over');
+    }); */
```
Comment (mimipeshy) [inline src/scripts/sorting.js]: - [ ] It’s better to delete commented code. If you need to review a previous version of your code, use git history.

**Review by mimipeshy — CHANGES_REQUESTED** [review]
## Hi @erezfree29 👋🏾 👋🏾 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
#### Highlights 

- A user can add, delete, and clear completed tasks ✔️
- A user can drag todo's to change the priority ✔️

- The updated array of items is stored in local storage ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

File: README.md [hunk 3]
```diff
@@ -20,8 +29,8 @@ the content in the webpage is produced almost purely by Javascript usage on a si
 
 To run the project on your local machine, please run the following commands on your terminal:
 
-> 1. run `git clone https://github.com/erezfree29/Restaurant/`
-> 2. run `cd Restaurant`
+> 1. run `git clone https://github.com/erezfree29/To_Do_List_2/`
+> 2. run `cd To_Do_List_2`
```
Comment (Dipeshtwis) [inline README.md:33]: [OPTIONAL]

- I would suggest to add the steps on how to run the project in local after cloning. You can mention some of the steps like mentioned below to guide the person who doesn't belong to technical field. :wink: 

```
npm install
npm start
```

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Dipeshtwis — APPROVED** [review]
# Hi @erezfree29, 

### Status: Approved ✔️

:+1: You have implemented the suggested changes. Well Done.

Your project is complete!  There are some optional changes I've suggested to you to make your project best, other than that nothing else to say.
Congratulations! 🎉 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- I would highly recommend to remove unnecessary files from your project. (Ex:- `.vscode`). You can put the file name into `.gitignore` to avoid pushing unnecessary file into Github.

**_Also check the comments under the review_**

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Dipesh Kumar_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉 🎉 

### Hi @erezfree29 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Keep Rocking!👏👏👏

**Pull request merged**